assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -84,17 +91,19 @@ function TextPane(props) {
       )}
 
       <Section title={__('Presets', 'web-stories')}>
```

Comment:
Is the `Section` title font size/family getting updated separately? It's slightly off from figma. 
![Screen Shot 2021-03-15 at 9 17 13 AM](https://user-images.githubusercontent.com/10720454/111185647-6d818180-856f-11eb-8db3-ce7f0c947206.png)
<img width="242" alt="Screen Shot 2021-03-15 at 9 17 23 AM" src="https://user-images.githubusercontent.com/10720454/111185649-6e1a1800-856f-11eb-950a-eec21371d9f3.png">

Reply:
Let's fix that here